tests/pact/verifier-test.repl

```diff
@@ -0,0 +1,43 @@
+(module m GOV
+  (defcap GOV () true)
+
+  (defcap GOOD ()
+    (enforce-verifier 'HYPERLANE)
+    )
+
+  (defcap BAD ()
+    (enforce-verifier 'HYPERLANE)
+    )
+
+  (defcap OUTERGOOD ()
+    (enforce-verifier 'HYPERLANE)
+    (compose-capability (GOOD))
+  )
+
+  (defun good ()
+    (with-capability (GOOD) 1)
+   )
+
+  (defun outergood ()
+    (with-capability (OUTERGOOD) 1)
+   )
+
+  (defun bad ()
+    (with-capability (GOOD)
+      (with-capability (BAD) 1)
+     )
+    )
+  )
+
+
+(env-verifiers [{"name":"HYPERLANE", "caps":[(OUTERGOOD)]}])
```

Comment:
Where is the payload that is being verified? Is that aspect of the SPI interface being tested? Also, if this is really an SPI -- meaning, an internal API that partners can use to extend Chainweb with new data verification schemes -- then I'd like to see a "canonical" such module implementing a basic verification scheme like ASCII which verifies that all data in the payload is 7-bit ASCII or something. Then partners could follow this canonical example by copying it and extending it as a starting point. That is missing from this PR, which makes it harder for me to see how it's an SPI from the integrator's point of view. So far, it reads like the just the plumbing necessary to create such an SPI, so far.

Reply:
Tests for the full verifier plugin interface including chainweb are going to be in chainweb, the only tests in here are the ones relevant to pact. pact doesn't actually care about anything in SPI besides the fact that it adds verifiers that are scoped to capabilities and can be required in caps with `enforce-verifier`. Info for partners wanting to integrate will be in both the KIP and the chainweb docs.